apps/addons/tests/test_models.py

```diff
@@ -2294,6 +2294,65 @@ def test_enable_addon(self, file_mock):
         assert not mock.hide_disabled_file.called
 
 
+class TestAddonWatchDeveloperNotes(amo.tests.TestCase):
+
+    def make_addon(self, **kwargs):
+        addon = Addon(type=amo.ADDON_EXTENSION, status=amo.STATUS_PUBLIC,
+                      **kwargs)
+        addon.save()
+        addon.versions.create(has_info_request=True)
+        addon.versions.create(has_info_request=False)
+        addon.versions.create(has_info_request=True)
+        return addon
+
+    def assertHasInfoSet(self, addon):
+        assert any([v.has_info_request for v in addon.versions.all()])
+
+    def assertHasInfoNotSet(self, addon):
+        assert all([not v.has_info_request for v in addon.versions.all()])
+
+    def test_has_info_save(self):
+        addon = self.make_addon()
+        self.assertHasInfoSet(addon)
+        addon.save()
+        self.assertHasInfoSet(addon)
+
+    def test_has_info_update_whiteboard(self):
```

Comment:
Same here, docstring explaining exactly what the test is testing. The test name does help, but I'd rather help our future selves remember exactly what this is about ;)
(Same for the tests below)